sample/allofhands_hawaii2016/session_types

```diff
@@ -0,0 +1,73 @@
+[
+
+        {   
+            "id": 1, 
+            "length": null, 
+            "name": "Welcome Reception"
+        },
+        {
+            "id": 2, 
+            "length": null, 
+            "name": "Breakfast"
+        }, 
+        {
+            "id": 3, 
+            "length": null, 
+            "name": "Lunch"
+        },
+        {
+            "id": 4, 
+            "length": null, 
+            "name": "Team Time"
+        }, 
+        {
+            "id": 5, 
+            "length": null, 
+            "name": "Team Time"
```

Comment:
All samples don't have any strict session types. What should I do in that case? It will be nice if you could suggest a way around @arp95 :)